### PR TITLE
Feature/change dto

### PR DESCRIPTION
**Pull request by C0Rs1K**

### CODE REVIEW

File: Curtains/Curtains.Application/DTO/BedspreadsDTO.cs [hunk 1]
```diff
@@ -25,21 +19,20 @@ public class BedspreadsDTO : BaseExtendedDTO
...
+        public int? FabricId { get; set; }
+        #endregion
+
+        #region NavigationProperties
         /// <summary>
-        ///  This property represents shape of bedspreads.
+        ///  This property represents size of bedspreads.
         /// </summary>
-        public string Shape { get; set; }
+        public BedspreadsSizeDTO Size { get; set; }
```
Comment (zd-prog) [inline Curtains/Curtains.Application/DTO/BedspreadsDTO.cs:20]: Why id is nullable, when Size model is not nullable?

File: Curtains/Curtains.Application/DTO/CurtainsDTO.cs [hunk 2]
```diff
@@ -18,13 +19,36 @@ public class CurtainsDTO : BaseExtendedDTO
         /// </summary>
         public int Width { get; set; }
         /// <summary>
+        ///  This property represents FabricId column.
+        /// </summary>
+        public int? FabricId { get; set; }
+        /// <summary>
+        ///  This property represents MaterialId column.
+        /// </summary>
+        public int? MaterialId { get; set; }
```
Comment (zd-prog) [inline Curtains/Curtains.Application/DTO/CurtainsDTO.cs]: Why this fields are nullable?

File: Curtains/Curtains.Application/DTO/CurtainsDTO.cs [hunk 3]
```diff
@@ -18,13 +19,36 @@ public class CurtainsDTO : BaseExtendedDTO
...
+        /// </summary>
+        public int? AccessoriesId { get; set; }
+        /// <summary>
+        ///  This property represents CurtainsTypeId column.
+        /// </summary>
+        public int? CurtainsTypeId { get; set; }
+        /// <summary>
+        ///  This property represents CurtainsKindId column.
+        /// </summary>
+        public int? CurtainsKindId { get; set; }
```
Comment (zd-prog) [inline Curtains/Curtains.Application/DTO/CurtainsDTO.cs]: same

**Review by zd-prog — CHANGES_REQUESTED** [review]
Check nullable fields, please

**Review by zd-prog — APPROVED** [review]

**Review by EgorAntonovich — APPROVED** [review]

**Pull request merged**